index.bs

```diff
@@ -5267,8 +5270,13 @@ complete</dfn> steps given |navigable| and |navigation status|:
       <pre class="cddl local-cddl">
         browsingContext.DownloadWillBegin = (
          method: "browsingContext.downloadWillBegin",
-         params: browsingContext.NavigationInfo
+         params: browsingContext.DownloadWillBeginParams
         )
+
+        browsingContext.DownloadWillBeginParams = {
+          suggestedFilename: text,
```

Comment:
If I am reading the HTML spec update correctly under certain cases this can be not present, should the CDDL here represent this as optional?

Reply:
I agree that this makes sense given the proposed HTML spec change.

Reply:
> I agree that this makes sense given the proposed HTML spec change.

@jgraham do you mean it makes sense to make the field optional: `?suggestedFilename: text`?

Reply:
No, sorry that was very unclear.

I mean that the spec as proposed makes sense i.e. afaict there's always some suggested filename.

Reply:
I build the PR to make it easier to read, and indeed the filename should always be present.